client/src/app/core/services/http.service.ts

```diff
@@ -33,14 +32,14 @@ export class HttpService {
         }
 
         const options = {
```

Comment:
This is not a change in this pull request, thus not to be changed here.

Reply:
You could change it in this PR nonetheless. It would save some work down the road if we change now everything we see. As something like "good practice". But I understand your point and will accept it.